<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFloodMessageDialog</class>
 <widget class="QDialog" name="AddFloodMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../assets/resources.qrc">
    <normaloff>:/icons/comment_16px.ico</normaloff>:/icons/comment_16px.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditMessage">
     <property name="placeholderText">
      <string>Message to flood</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButtonPublic">
     <property name="text">
      <string>Public</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxPublic">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxCanal">
        <item>
         <property name="text">
          <string>General</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Guild</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aliance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Team</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Recruitment</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Recipient of messages</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Interval between messages</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="currentSection">
         <enum>QDateTimeEdit::SecondSection</enum>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>10</second>
         </time>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radioButtonPrivate">
     <property name="text">
      <string>Private</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxPrivate">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxSomeonEntersMap">
        <property name="text">
         <string>When someone enters the map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxSomeoneLeavesMap">
        <property name="text">
         <string>When someone leaves the map</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../assets/resources.qrc">
         <normaloff>:/icons/plus_16px.ico</normaloff>:/icons/plus_16px.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../assets/resources.qrc">
         <normaloff>:/icons/cancel_16px.ico</normaloff>:/icons/cancel_16px.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBoxPublic</zorder>
  <zorder>lineEditMessage</zorder>
  <zorder>groupBoxPrivate</zorder>
  <zorder>radioButtonPublic</zorder>
  <zorder>radioButtonPrivate</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditMessage</tabstop>
  <tabstop>radioButtonPublic</tabstop>
  <tabstop>comboBoxCanal</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>radioButtonPrivate</tabstop>
  <tabstop>checkBoxSomeonEntersMap</tabstop>
  <tabstop>checkBoxSomeoneLeavesMap</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
